rules/group.go

```diff
@@ -42,6 +42,14 @@ import (
 	"github.com/prometheus/prometheus/tsdb/chunkenc"
 )
 
+type GroupState int
```

Comment:
Do you mind adding comment on this exported struct?

Reply:
I'm going to make this struct private. After looking at this again, I don't see why this has to be exported